package.json

```diff
@@ -36,7 +36,7 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
-    "@angular/cli": "^1.0.0",
+    "@angular/cli": "^1.0.2",
```

Comment:
@nikhilrayaprolu Should I revert back this too? I have deleted the `TopbarComponent`. Almost finished. :smile: